mention-bot.js

```diff
@@ -9,7 +9,17 @@
  * @flow
  */
 
-var downloadFileSync = require('download-file-sync');
+var githubWebClient = require('./githubWebClient');
+
+var downloadFileSync = function(url, cookies) {
+  var command = 'curl -L ' + url;
```

Comment:
It was giving me trouble with cookies and `execFileSync` but I can look into it more.

The array I ended up passing was somethign like:

```js
[ '-L',
  'https://github.com/private/repo/pull/2053.diff',
  '-H',
  '"Cookie: logged_in=yes;  domain=.github.com;  path=/;  expires=Wed, 21 Nov 2035 16:47:48 -0000;  secure;  HttpOnly; _gh_sess=__SESS__; user_session=__USERSESS__; dotcom_user=zekenie"' ]
```

But github was giving me a 500 error. So I tried running it myself  with curl and it worked fine. Then I switched it to regular `execSync` and it worked. Any suggestions? I can work on hunting it down further.